### PR TITLE
[Audio] Better error handling after multiple subsequential errors

### PR DESCRIPTION
**Pull request by Drapersniper**
### Type

- [X] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes

Auto disconnects the player if there are more than 5 errors in the last 10 seconds. (Likely caused by ... YouTube or playing local album with broken tracks etc)

Benefit of this is to reduce message spam caused by the bot when letting them know the playback broke

Also while at it fixed 1 bug where the DJ command was crashing

### CODE REVIEW

**Review by PredaaA — DISMISSED** [review]
I tested this PR and works as intended.

**Review by mikeshardmind — DISMISSED** [review]

**Comment by mikeshardmind** [comment]
Needs conflict resolution.

**Comment by Drapersniper** [comment]
Conflicts resolved

File: redbot/cogs/audio/audio.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def __init__(self, bot):
             status=False,
             use_external_lavalink=False,
             restrict=True,
-            current_version=redbot.core.VersionInfo.from_str("3.0.0a0").to_json(),
+            current_version=redbot.core.VersionInfo.from_str(__version__).to_json(),
```
Comment (mikeshardmind) [inline redbot/cogs/audio/audio.py]: What is this value being used for right now?

If it is or has been used for migrating data, this is not a correct change and will break anyone upgrading from the 3.0.0 -> 3.2.0 as the default here should be the lowest version, with migrations setting the version related to their migration.

Comment (Drapersniper) [reply redbot/cogs/audio/audio.py]: afaik this is not currently used for anything ... but @aikaterna  should be able to confirm it already existed before I touched audio..

**Comment by Drapersniper** [comment]
Do not merge this one ... Merge #3205

File: redbot/cogs/audio/audio.py (hunk 1 above)
Comment (Drapersniper) [reply redbot/cogs/audio/audio.py]: This is unused anywhere within red ... I've remove this default

**Comment by Drapersniper** [comment]
Will setup a CPR for all of this so itsd easier to follow ... less likely to fuck up)

**Pull request closed without merging**